components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "defectdojo",
-  "version": "2.30.4",
+  "version": "3.31.0",
```

Comment:
2.31.0 ?

Reply:
Ohhh boy :(

Reply:
I believe this to be corrected:
- versions have been updated on master in all the places
- docker images have been republished with 3.x versions removed
- helm chart has been republished
- bugfix and dev left alone as they have `2.32.0-dev` versions